tests/data/eqpt_config.json

```diff
@@ -75,8 +84,8 @@
             "target_pch_out_db": -20,
             "add_drop_osnr": 38,
             "restrictions": {
-                            "preamp_variety_list":["low_gain_preamp", "high_gain_preamp"],
```

Comment:
"low_gain_preamp" and "high_gain_preamp" where probably given as examples names but these amp where not defined in the edfa list. Since the feature was not coded, there was not effect or impact.  Now that I have coded and activated the feature there should be valid names here. But in this case this will change many tests. So I prefer to leave it empty for now, and to create a specific test for restrictions later on.

Reply:
Ah, right, thanks. It was not implemented before :).